### ISSUE TITLE
Web - Can Not Open "Share" Button with Keyboard

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#55]
Can't open the "Share" button/link because focus goes from the "Search" button/link to the "About" button/link. While tabbing through the footer section, I'm not able to tab to the "Share" button/link. The focus goes from the "Search" to the "About" buttons/links. User shouldn't skip over an item like the "Share" button/link with a keyboard.

**Comment by KyleLivingstonDOL** [comment]
Thanks John. ZTI, can we fix this?

**Comment by jfc3-dol** [comment]
Not able "Share" button/link in the footer with a keyboard. There seems to be a whole keyboard focus issue going on depending on what page you are on and can't open things with a keyboard, or the wrong thing is opened because visual and keyboard focus doesn't match for some reason.

**Comment by Abychalil** [comment]
fixing in progress

**Comment by jfc3-dol** [comment]
@Abychalil 
I can now open the "Share" button with a keyboard in Chrome and Firefox but not with IE 11, and Edge is not even displaying the home page.

**Comment by jfc3-dol** [comment]
@Abychalil

I'm not able to open the "Share" button in the footer with IE 11 and Edge. I can now open the "Share" button with a keyboard in Chrome and Firefox.

**Comment by jfc3-dol** [comment]
@harpreetsdev @Abychalil @pbhatt17

I am going to open a new issue for the specific issue related to IE 11 and Edge for JIRA issue CCP-134.

Once that is done, I will add the GitHub issue here and then close it.

**Comment by jfc3-dol** [comment]
@harpreetsdev @Abychalil @pbhatt17

Opened new issue #146, so going to close this one.
